### PR TITLE
Update version to 16.8

### PR DESCRIPTION
**Pull request by nikhilem**
- Update version in `site-constants`
- Add 16.8 entry in `content/versions.yml`

### CODE REVIEW

**Comment by nikhilem** [comment]
Please feel free to add link to React 16.7 documentation in the versions page

File: content/versions.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+- title: '16.8'
+  changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#1680-february-6-2019
 - title: '16.7'
```
Comment (bvaughn) [inline content/versions.yml]: We'll want to add a `path` and `url` for the 16.7 docs too. I can help with that bit.

Comment (nikhilem) [reply content/versions.yml]: Feel free to add them :)

**Comment by bvaughn** [comment]
Thanks again :)

**Pull request merged**